dev/drivers/scripts/prep/glwu/jevs_glwu_wave_grid2obs_prep.sh

```diff
@@ -0,0 +1,74 @@
+#PBS -N jevs_glwu_wave_grid2obs_prep
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=00:15:00
+#PBS -l select=1:ncpus=1:mem=5GB
+#PBS -l debug=true
+
+
+set -x
+
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+
+export MODELNAME=glwu
+export NET=evs
+export COMPONENT=glwu
```

Comment:
Using this setup, it looks like "glwu" will appear twice in the same directory path. Is that true? Can one of you please copy and paste the COMOUT path for glwu prep data? 

@SamiraArdani-NOAA @malloryprow

Reply:
`grep "COMOUT" jevs_glwu_wave_grid2obs_prep.o188268619` 

> export COMOUT=/lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr527/evs/v2.0/prep/glwu/wave
> COMOUT=/lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr527/evs/v2.0/prep/glwu/wave
> export COMOUT=/lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr527/evs/v2.0/prep/glwu/wave
> COMOUT=/lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr527/evs/v2.0/prep/glwu/wave
> export COMOUT=/lfs/h2/emc/vpppg/noscrub/mallory.row/verification/EVS_PRs/pr527/${NET}/${evs_ver_2d}/${STEP}/${COMPONENT}/${RUN}

Reply:
Thanks! For prep, it's important that the path is `/glwu/wave` for NCO's scrubbers. For stats and plots, it's also important for NCO's HPSS archival scripts!